docs/index.md

```diff
@@ -67,6 +67,14 @@ When you have your reciped created and saved on your disk, you can execute your
 zampy /path_to_recipe/sample_recipe.yml
 ```
 
+>NOTE: You may recieve an error message from CDS/ADS if not all the required
+>licences have been accepted. Follow the instructions in the error message to
+>accept the licences and run the recipe again.
```

Comment:
```suggestion
!!! note

    You may recieve an error message from CDS/ADS if not all the required
    licences have been accepted. Follow the instructions in the error message to
    accept the licences and run the recipe again.
```
mkdocs supports [admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/). Probably looks nicer and more clear for the user